notify/android/NotificationHelper.java

```diff
@@ -34,7 +34,7 @@ public static void sendNotification(Context ctx, String channelID, int notificat
                 .setSmallIcon(Icon.createWithBitmap(bitmap))
                 .setContentText(text)
                 .setContentIntent(pending)
-                .setPriority(Notification.PRIORITY_DEFAULT);
+                .setOngoing(onGoing);
```

Comment:
The setPriority line was removed because it's deprecated and properly handled in the notification channel, I'm guessing whoever switched to the new system forgot to remove it